tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
+envlist = flake8, {py26,py27,py33,py34}-dj{16,17}-cms{30,31}
```

Comment:
Please use:

`flake8, py{34,27}-dj{18,17}-cms{31,30}, {py27,py26}-dj16-cms{31,30}`

This avoids configurations that won't work with one another.

Then, you can create your envlist for use in .travis.yml with:

````
tox -l
````

Which yields:

````
py34-dj18-cms31
py34-dj18-cms30
py34-dj17-cms31
py34-dj17-cms30
py27-dj18-cms31
py27-dj18-cms30
py27-dj17-cms31
py27-dj17-cms30
py27-dj16-cms31
py27-dj16-cms30
py26-dj16-cms31
py26-dj16-cms30
````